packages/types/rollup.config.js

```diff
@@ -0,0 +1,56 @@
+import fs from 'node:fs';
+import ts from 'typescript';
+import del from 'rollup-plugin-delete';
+import resolve from '@rollup/plugin-node-resolve';
+import typescript from 'rollup-plugin-typescript2';
+import { babel } from '@rollup/plugin-babel';
+import process from 'process';
+
+const extensions = ['.js', '.jsx', '.ts', '.tsx'];
+const packageJson = JSON.parse(fs.readFileSync('package.json', 'utf-8'));
+const { dependencies, peerDependencies } = packageJson;
```

Comment:
1.I do not see any other use of packageJson here so you can directly destroy
```suggestion
const { dependencies, peerDependencies } = JSON.parse(fs.readFileSync('package.json', 'utf-8'));
```

2. better to use optional default values in case you will remove dependencies or peerDependencies later